config/testgrids/conformance/conformance-all.yaml

```diff
@@ -219,6 +220,12 @@ dashboards:
       description: Runs conformance tests using kubetest against kubernetes v1.17 with cloud-provider-huaweicloud
       test_group_name: cloud-provider-huaweicloud-e2e-conformance-release-v1.17
 
+- name: conformance-cloud-provider-baiducloud
+  dashboard_tab:
+    - name: Baidu Cloud Provider, v1.13
```

Comment:
I'm not going to block this right now as I don't think we have policy, but 1.13 is way out of support upstream and cannot be certified for conformance currently

Reply:
we were really mostly hoping for people to publish tests against the latest kubernetes code when we set up the program for people to publish conformance results to testgrid :disappointed: